docs/get.md

```diff
@@ -0,0 +1,19 @@
+# Get
+
+The get command supports the following sub-commands:
+
+* [recovery-profile](#recovery-profile)
+
+## recovery-profile
+
+This command will display the recovery-profile value set for the osd.
+
+Example
+
+```bash
+odf get recovery-profile
```

Comment:
What about combining the command with the output in a single block? But if you prefer how it was, you can ignore this.
```suggestion
$ odf get recovery-profile
high_recovery_ops
```

Reply:
I think this make sense